packages/account/src/Components/address-details/address-details.tsx

```diff
@@ -281,13 +296,13 @@ const AddressDetails = observer(
                                                     placeholder={localize('State/Province')}
                                                     disabled={
                                                         disabled_items.includes('address_state') ||
-                                                        (props.value?.address_state && has_real_account)
+                                                        (!!props.value?.address_state && has_real_account)
                                                     }
                                                 />
                                             )}
                                             <FormInputField
                                                 name='address_postcode'
-                                                required={is_gb_residence}
+                                                required={!!is_gb_residence}
```

Comment:
I don't think this line of code is required as we do not support GB accounts anymore, please have a look 🙏🏼